middleware/configuration/include/configuration/user.h

```diff
@@ -658,6 +658,7 @@ namespace casual
                std::optional< std::string> alias;
                std::optional< std::string> queuebase;
                std::optional< std::string> note;
+               std::optional< std::string> capacity;
```

Comment:
I think I'd rather have. 
```yaml
group:
   capacity:
      size: 42
      # we could add messages or something. Not likely maybe...
```
or
```yaml
group:
   limit:
      size: 42
      # or capacity: 42
```

or
```yaml
group:
   size:
      capacity: 42
```
or we keep it as is, and if we ever want to add other stuff we deprecate `capacity` and create som composite as above. Your call.